README.md

```diff
@@ -219,6 +221,40 @@ functions:
 
 For more info on `serverless.yml` syntax, see their docs.
 
+#### securitySchemes
+
+You can provide optional Security Schemes:
+
+```yml
+custom:
+  documentation:
+    securitySchemes:
+      my_api_key:
+        type: apiKey
+        name: api_key
+        in: header
+```
+
+It accepts all available Security Schemes and follows the specification: https://spec.openapis.org/oas/v3.0.3#security-scheme-object
+
+#### Security on each operation
+
+To apply an overall security scheme to all of your operations without having to add the documentation to each one, you can write it like:
+
+```yml
+custom:
+  documentation:
+    securitySchemes:
+      my_api_key:
+        type: apiKey
+        name: api_key
+        in: header
+    overallSecurityRequirement:
```

Comment:
Why not stick to standard naming "security" instead of overallSecurityRequirement. We know that it should be applied to all paths because it is in the root of the documentation object. If it was path specific it would have been defined inside of the path object.

Reply:
OpenAPI does this by itself, if it's defined in the root it will be applied, no need to copy from root to each path

Reply:
> OpenAPI does this by itself, if it's defined in the root it will be applied, no need to copy from root to each path

Not 100% sure what you mean here.